packages/plugin-video/web/src/toolbar/video-settings.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { useState } from 'react';
+import { VideoSettingsProps } from '../types';
+import classNames from 'classnames';
+import { mergeStyles } from 'wix-rich-content-common';
+import Styles from '../../statics/styles/video-settings.scss';
+import {
+  SettingsSection,
+  LabeledToggle,
+  SettingsPanelFooter,
+  SettingsMobileHeader,
+} from 'wix-rich-content-plugin-commons';
+
+const VideoSettings: React.FC<VideoSettingsProps> = ({
+  componentData,
+  helpers,
+  pubsub,
+  theme,
+  t,
+  isMobile,
+  settings,
+}) => {
+  const disableDownload = settings.disableDownload || componentData?.config?.disableDownload;
```

Comment:
config is generally for the wrapper data (alignment, size...). In the new schema we will stick to this better. 
```suggestion
  const disableDownload = settings.disableDownload || componentData?.disableDownload;
```